backend/codingchallenge/api/include/githubApi.py

```diff
@@ -72,43 +80,46 @@ def addLinter(self, repoName):
             return self.gApi.get_organization(self.githubOrg).get_repo(repoName).create_file(
                 path=".github/workflows/megalinter.yml",
                 message="added megalinter",
-                content=open(BASE_DIR.joinpath("api/include/megalinter.yml"), 'r').read())
+                content=open(BASE_DIR.joinpath("api/include/megalinter.yml"), 'r').read()
+            )
 
     def getLinterLog(self, repoName):
         if settings.DEPLOY_OFFLINE:
             return githubApiMockData.getLinterLog
         else:
             return self.gApi.get_organization(self.githubOrg).get_repo(repoName).get_contents(
-                'megalinter-reports/megalinter.log').decoded_content.decode()
+                'megalinter-reports/megalinter.log'
+            ).decoded_content.decode()
 
     def getLinterResult(self, repoName):
-        if settings.DEPLOY_OFFLINE:
```

Comment:
Offline Deployment possibility got removed by this change. -> add again

Reply:
In the Code above is no call that has to be secured. It only calls getLinterLog which has the DeployOffline Option

Reply:
@Marcodaum Resolving Merge Conflicts resolved this too.